pylint/checkers/variables.py

```diff
@@ -3046,7 +3046,10 @@ def _check_module_attrs(
     def _check_all(
         self, node: nodes.Module, not_consumed: dict[str, list[nodes.NodeNG]]
     ) -> None:
-        assigned = next(node.igetattr("__all__"))
+        try:
+            assigned = next(node.igetattr("__all__"))
```

Comment:
Why don't we use `safe_infer` here?

Reply:
Do you know of a way to use a helper with `igetattr` (or to use something besides igetattr, here)?

Reply:
Ah, of course.. Well, seems like we need `safe_igetattr` 😄